api/catalog/api/utils/pagination.py

```diff
@@ -42,6 +45,8 @@ def page(self, value):
         value = int(value)  # convert str params to int
         if value <= 0:
             raise get_api_exception("Page must be greater than 0.", 400)
+        elif value > 20:
+            raise get_api_exception("Searches are limited to 20 pages.", 400)
```

Comment:
This validation should be removed from here and the new validation should be added to the serializer: https://github.com/WordPress/openverse-api/blob/d26df92f04120e7382d042ae0b439fb6ea9209be/api/catalog/api/serializers/media_serializers.py#L167-L174

```suggestion
```

It was erroneously added back to this area during the revert.

Reply:
Actually, this is inaccurate. The validation needs to be moved to the serializer, but I think it should be done in a different PR dedicated to a small refactor of the pagination parameter validation. We're under-utilising serializer validation at the moment and we can delete a significant amount code. But it should be done in a different PR.